tox/config.py

```diff
@@ -9,6 +9,7 @@
 import pkg_resources
 import itertools
 import pluggy
+from subprocess import list2cmdline
```

Comment:
yes, but it works in py26 and py35 so that's ok.  also subprocess module does mention it IIRC.